src/install_host_app.sh

```diff
@@ -126,17 +127,15 @@ echo "Installing $BROWSER_NAME host config"
 mkdir -p "$TARGET_DIR"
 
 PATH_ESC="$(echo "$PATH" | sed -e 's/@/\\@/g')"
-PASS_PATH_ESC="$(echo "$PASS_PATH" | sed -e 's/@/\\@/g')"
+PATH_HOMED="$(echo "$PATH_ESC" | sed -e "s@$HOME@~@g")"
```

Comment:
This will not work if `$HOME` contains an `@` char.

Reply:
Okay fair enough, but that's easily solved. I'll submit an updated PR as soon as we resolved the above.